packages/cozy-client/src/CozyClient.js

```diff
@@ -1193,7 +1193,7 @@ client.query(Q('io.cozy.bills'))`)
           if (queryDef instanceof QueryDefinition) {
             definitions.push(queryDef)
           } else {
-            documents.push(queryDef)
+            documents.push(doc)
```

Comment:
Is this part of the fix? I don't understand the impact compared to the previous implementation